app/grants/views.py

```diff
@@ -387,13 +387,20 @@ def grant_fund(request, grant_id, grant_slug):
             subscription.contributor_profile = profile
             subscription.grant = grant
             subscription.save()
+            activity_kwargs = {
```

Comment:
it'd be nice if the `activity_kwargs` and `Activity.objects.create` code was DRY'd.  maybe refactoring it into a `record_grant_activity_helper(activity_type, subscription)` would suffice

Reply:
OK